orttraining/orttraining/core/optimizer/megatron_transformer.cc

```diff
@@ -113,6 +113,21 @@ static uint32_t HashName(const std::string& name) {
   return hash;
 }
 
+static bool FilterConsumeNodeWithOpType(const Node& node) {
```

Comment:
can you rename this function to specify that it is checking for MatMul and/or add a "MatchesOpType" function to graph_utils?